assets/dev/js/editor/utils/controls-css-parser.js

```diff
@@ -37,6 +37,33 @@ ControlsCSSParser = elementorModules.ViewModule.extend( {
 		} );
 	},
 
+	updateBackgroundImagesSizes( control, values ) {
+		if ( control.name.includes( 'background_image' ) ) {
+			const imageSize = values.background_dimension || false;
+			if ( imageSize ) {
+				let responsiveSize = 'background_dimension';
+				const breakpoints = elementorFrontend.config.responsive.breakpoints;
+				Object.entries( breakpoints ).forEach( ( [ breakpointName ] ) => {
+					if ( control.name.includes( breakpointName ) ) {
+						responsiveSize += '_' + breakpointName;
+					}
+				} );
```

Comment:
I think you can get the breakpoint that's associated with the control without iterating over all of them like this. 
Maybe consult with @udidol

Reply:
I need to support all types of breakpoints. In the future also with custom breakpoints.

Reply:
You don't need to iterate over these breakpoints to get the responsive control name.

Each responsive control has a `responsive` property that you can use to get the prefix. Specifically, we already have an existing method that does exactly what you need:
`elementor.conditions.getResponsiveControlDeviceSuffix( control.responsive )`